### DISCUSSION TITLE
Galaxy Brain Achievement Help

### DISCUSSION THREAD

**Discussion by peripheralmike** [Schweinepriester/github-profile-achievements#660] [Q&A]
Please help me gain the Galaxy Brain Achievement.

**Comment by AnzhiZhang** [comment]
#663 please

**Comment by LauraBigoni** [comment]
For galaxy brain you need to suggest answers to questions posted in discussion. If your answer is helpful then the person who asked the question can mark you answer as the solution. Once you've had 2 questions marked as solutions then you'll get galaxy brain.

Here's the link to the general GitHub community discussions. It might be a good place to start. https://github.com/community

If you found this useful then please mark my answer as a solution so I can get the badge.

**Comment by PARKER-X** [comment]
You need to answer of 2 questions of disccussion.
